packages/storybook/.storybook/preview.tsx

```diff
@@ -23,6 +23,7 @@ registerIcons({ icons: DEFAULT_COMPONENT_ICONS });
 
 export const parameters = {
   layout: 'fullscreen',
+  previewTabs: { 'storybook/docs/panel': { index: -1 } },
```

Comment:
noticed when I go to a page from overview, I'm still take to the canvas (not sure if there's any way to overcome that)

Reply:
There is some stickyness between the components. I will look into seeing how to reset the selection